Estore.UITests/Features/CreateUsersTests.feature

```diff
@@ -0,0 +1,136 @@
+﻿Feature: CreateUsersTests
+
+As a Admin
+I would like to create new customer via UI
+
+#US_49_25a
+@AdminLoggedIn
+Scenario: US_49_25a_Admin abserve the layout and elements on the Create user modal window
+	Given Admin click on the Users button
+	And User page is open
+	When Admin click on the Add User button
+	Then Create user modal window should be open
+	And Create user modal window contains fields: 'First name, Last name, Email, Password, Password'
```

Comment:
use array of string instead of splitting string

Reply:
fixed